VersionTracker/VersionTracker.xcodeproj/project.pbxproj

```diff
@@ -290,7 +290,6 @@
 				E2D4BF241D1E878F007A4E68 /* Frameworks */,
 				E2D4BF251D1E878F007A4E68 /* Headers */,
 				E2D4BF261D1E878F007A4E68 /* Resources */,
-				E2D4BFC31D1E8AFB007A4E68 /* Run Script - ⚠️ SwiftLint */,
```

Comment:
Why is it necessary? What error did you get? 
The script is still useful, I rather prefer to keep it